cgi-bin/DW/External/User.pm

```diff
@@ -69,13 +70,17 @@ sub ljuser_display {
     my $display_class = $opts{no_ljuser_class} ? "" : " class='ljuser'";
     my $domain = $self->site->{domain} ? $self->site->{domain} : $self->site->{hostname};
 
+    $nolink = 1 if exists $DW::External::Site::deadsites{$domain};
```

Comment:
I would suggest combining this line and the initial nolink declaration as follows:

`my $nolink = exists $DW::External::Site::deadsites{$domain} ? 1 : 0;`

This is called the ternary operator 😄

Reply:
Since we're now using a `get_deadsites` function, this syntax needs to be updated:

```
my %deadsites = DW::External::Site::get_deadsites();
my $nolink = exists $deadsites{$domain} ? 1 : 0;
```

Sorry for not noticing that in my previous review! I **think** that's the last update necessary for this to work.

I would also highly encourage you to make sure your Perl edits are tidy going forward. @momijizukamori posted a pre-commit hook in dw-dev recently to help make this easier!